lang/en.js

```diff
@@ -10,9 +10,9 @@ export default {
 	"components.form-element.defaultError": "{label} is invalid.",
 	"components.form-element.defaultFieldLabel": "Field",
 	"components.form-element.input.email.typeMismatch": "Email is not valid",
-	"components.form-element.input.number.rangeError": "Number must be higher than{minInclusive} {min} and lower than{maxInclusive} {max}.",
-	"components.form-element.input.number.rangeOverflow": "Number must be lower than{maxInclusive} {max}.",
-	"components.form-element.input.number.rangeUnderflow": "Number must be higher than{minInclusive} {min}.",
+	"components.form-element.input.number.rangeError": "Number must be {minExclusive, select, true {greater than} false {greater than or equal to}} {min} and {maxExclusive, select, true {less than} false {less than or equal to}} {max}.",
+	"components.form-element.input.number.rangeOverflow": "Number must be {maxExclusive, select, true {less than} false {less than or equal to}} {max}.",
+	"components.form-element.input.number.rangeUnderflow": "Number must be {minExclusive, select, true {greater than} false {greater than or equal to}} {min}.",
```

Comment:
We'll need to be extra careful when the translations for this come back to ensure they get translated properly. Can you double-check that for a different language which doesn't have the extra replacements that it at least doesn't blow up by passing in the minExclusive/maxExclusive value?

Reply:
Tested it with `fr` and nothing seems to blow up